ui/store/store.ts

```diff
@@ -0,0 +1,132 @@
+import { StoreEnhancer } from 'redux';
+import { configureStore as baseConfigureStore } from '@reduxjs/toolkit';
+import devtoolsEnhancer from 'remote-redux-devtools';
+import { ApprovalControllerState } from '@metamask/approval-controller';
+import { GasEstimateType, GasFeeEstimates } from '@metamask/gas-fee-controller';
+import rootReducer from '../ducks';
+import { LedgerTransportTypes } from '../../shared/constants/hardware-wallets';
+import { TransactionMeta } from '../../shared/constants/transaction';
+
+/**
+ * This interface is temporary and is copied from the message-manager.js file
+ * and is the 'msgParams' key of the interface declared there. We should get a
+ * universal Message type to use for this, the Message manager and all
+ * the other types of messages.
+ *
+ * TODO: Replace this
+ */
+export interface TemporaryMessageDataType {
+  id: number;
+  type: string;
+  msgParams: {
+    metamaskId: number;
+    data: string;
+  };
```

Comment:
is there other keys like `origin`, `from`, `version` missing in this type?

Reply:
Yes, I aimed for the minimal typing because we want to make this type more robust when we hit the background data layer, this was like MVP for getting types that work in actions.ts

Reply:
gotcha!